papers/templates/papers/paper.html

```diff
@@ -146,6 +146,20 @@ <h1 id="paperTitle" data-params="{csrfmiddlewaretoken:'{{csrf_token}}'}" data-pk
         <div class="detailsContent"><div class="abstract">{% autoescape off %}{{ paper.abstract }}{% endautoescape %}</div></div>
     {% endif %}
 
+    {% with doi=paper.get_doi  %}
+        {% if doi and not paper.pdf_url %}
+            <h2>{% trans "Not the author?" %}</h2>
+            <p>
+                {% blocktrans count author_nb=paper.author_count %}
+                    Encourage the author to liberate this publication with the Open Access Button.
+                    {% plural %}
+                    Encourage the authors to liberate this publication with the Open Access Button.
+                {% endblocktrans %}
+            </p>
+            <p><a href="https://openaccessbutton.org/request?doi={{ doi }}" target="_blank" class="btn btn-default btn-xs"><img src="{% static "img/icon_OAB.png" %}" style="height:20pt"> {% trans "Send request" %} <img src="{% static "extlink.png" %}" alt="[^]"></a></p>
```

Comment:
You can use `btn-primary` here instead of `btn-default` to make it blue I think. See https://getbootstrap.com/docs/3.3/components/#btn-dropdowns-single for instance.

Do you know if OAB has a specific policy for styling their logo?

Reply:
Thanks!

Their site content is CC-BY. The only thing happening here is a resize (in the browser), that's fine.